File: Hashing/basic_hashing.py
n = int(input())
arr =  [int(input()) for _ in  range(n)]
hash_arr = [0]*(n+1)
for i in range(n):
    hash_arr[arr[i]] =hash_arr[arr[i]] + 1

query = int(input("Enter the queery")) 
print(hash_arr[query])

File: Hashing/basic_hashing.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    int  n; 
    cin>> n;
    int arr[n];
    int hash[n+1] = {0};
    for (int i=0;i<n;i++){
        cin >> arr[i];
        hash[arr[i]] = hash[arr[i]]+ 1;
    }
    int q ;

    cout << "VALUE of q \n";
    cin >> q ;
    while (q--){
    int query;
    cout<< "\nAsking Query -----";
    cin >> query;
    cout << "   " << hash[query];
        }
    return 0;
}
